jobs/build/buildvm-maint/Jenkinsfile

```diff
@@ -46,9 +51,9 @@ node('openshift-build-1') {
             try {
                 stage("push images") {
                     dir ( "enterprise-images" ) {
-                        sh 'doozer --group sync-3.9 images:push --to-defaults'
-                        sh 'doozer --group sync-misc images:push --to-defaults'
-                        sh 'doozer --group sync-3.7 images:push --to-defaults'
+                        sh 'doozer --working-dir ${doozer_working} --group sync-3.9 images:push --to-defaults'
```

Comment:
Add `buildlib.cleanWorkdir(doozer_working)` between each of these calls.
Otherwise, looks good.

Reply:
thanks! done